sentinel-demo/sentinel-demo-zuul2-gateway/src/main/java/com/alibaba/csp/sentinel/demo/zuul2/gateway/ZuulBootstrap.java

```diff
@@ -51,7 +52,7 @@ public void start() {
             server = serverStartup.server();
             server.start(true);
         } catch (IOException e) {
-            e.printStackTrace();
+            RecordLog.warn("[ZuulBootstrap] Error when start", e);
```

Comment:
Maybe it's not a good idea to print it to logs in demo.

Reply:
yes. I ignored it in test and demo. this one I will remove it.